tests/main_functions.lua

```diff
@@ -0,0 +1,51 @@
+local luaclang = require "luaclang"
```

Comment:
I would move this file to the root and call it `clang_spec.lua`.

Reply:
I think dedicating a separate folder for tests is a better idea. It will be needed later on, when we need separate input files and also the tests will be modular @gligneul

Reply:
I think we only need one spec file, but given that you have input files, you should at least rename the directory name so busted can find your tests automatically.
```
spec/clang_spec.lua
```
Also, I would not create a `input` subdirectory, it seems overkill for now.